### DISCUSSION TITLE
Our YOURLS account stopped working

### DISCUSSION THREAD

**Discussion by jamaaludeen786** [YOURLS/YOURLS#3071] [Q&A] [answered]
Hi

Our YOURLS account has stopped working. All the shortened URLs that we've created redirects to http://seeke.rs/cgi-sys/suspendedpage.cgi

I can provide the username and password if required. 

Please advise on the steps we need to take to resolve this. 

Thank you
Jamaaludeen

**Comment by dgw** [comment, marked as answer]
**You should never provide the login credentials for your YOURLS account to a stranger, but especially not in a public forum like this.**

That said: The YOURLS project does not offer a "YOURLS account" or hosting feature, free or paid. This is a problem that needs to be resolved with your hosting company, which appears to be BlueHost.

![image](https://user-images.githubusercontent.com/164140/134400583-f2a2955c-d587-43c6-9999-0c792fc05935.png)
![image](https://user-images.githubusercontent.com/164140/134400624-b086c28e-cb85-4a45-9638-9bfcaf436f51.png)
